.github/workflows/docker-publish.yml

```
@@ -1,6 +1,8 @@
 name: Build & Push
 
 on:
+  push:
+    branches: [main]
   pull_request:
     branches: [main]
   tags: ["v*"]
```

actionlint (1.7.11)

[error] unknown Webhook event "tags". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)